cmp/grid/columns/Column.ts

```diff
@@ -1000,26 +1000,20 @@ export class Column {
         return sortCfg ? sortCfg.comparator(v1, v2) : GridSorter.defaultComparator(v1, v2);
     };
 
-    private sortToBottomComparator = (v1: any, v2: any, sortDir: 'asc' | 'desc') => {
-        const {sortToBottom} = this,
-            asc = sortDir === 'asc';
+    private sortToBottomComparator = (v1, v2, sortDir: 'asc' | 'desc') => {
+        const {sortToBottom} = this;
 
-        if (isEmpty(sortToBottom)) return 0;
+        if (isNil(sortToBottom)) return 0;
 
-        let v1ToBottom = null,
-            v2ToBottom = null;
-
-        sortToBottom.forEach((it, idx) => {
-            const fn = isFunction(it) ? v => it(v) : v => v === it;
-            if (isNil(v1ToBottom) && fn(v1)) v1ToBottom = idx;
-            if (isNil(v2ToBottom) && fn(v2)) v2ToBottom = idx;
-        });
-
-        if (!isNil(v1ToBottom) && !isNil(v2ToBottom)) {
-            return (v1ToBottom - v2ToBottom) * (asc ? 1 : -1);
+        for (let fn of sortToBottom) {
+            const v1ToBottom = fn(v1),
+                v2ToBottom = fn(v2);
+            const isAsc = sortDir === 'asc';
+            if (v1ToBottom != v2ToBottom) {
+                return v1ToBottom ? (isAsc ? 1 : -1) : isAsc ? -1 : 1;
+            }
```

Comment:
I don't understand how this accounts for the order of entries in sortToBottom? I'm talking about the case when both values return true for _different_ closures within the config, and we want to respect the order of the closures.